messenger/message-recorder.rst

```diff
@@ -0,0 +1,120 @@
+.. index::
+    single: Messenger; Record messages; Transaction messages
+
+Transactional Messages: Handle Events After CommandHandler Is Done
+==================================================================
+
+A message handler can ``dispatch`` new messages during execution, to either the same or
+a different bus (if the application has `multiple buses </messenger/multiple_buses>`_).
+Any errors or exceptions that occur during this process can have unintended consequences,
+such as:
+
+- If using the ``DoctrineTransactionMiddleware`` and a dispatched message to the same bus
+  and an exception is thrown, then any database transactions in the original handler will
+  be rolled back.
+- If the message is dispatched to a different bus, then the dispatched message can still
+  be handled even if the original handler encounters an exception.
```

Comment:
"can still be handled"? Are you saying "the dispatched message will still be handled (which is good) even if the original handler has an exception" or "the dispatched message will still be handled (which is bad) even if the original handler has an exception". I can't tell what the current behavior vs desired behavior is.